src/functions/docker-exec-util.go

```diff
@@ -0,0 +1,235 @@
+package functions
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
```

Comment:
This really is a dup of line 20 import. By pulling in `github.com/docker/docker/api/types/containers` you are already pulling in `github.com/docker/docker/api/types`

Reply:
@markyjackson-taulia I was reading and trying to remove these dup. But I found that this is the standard followed by Golang. The types will have their own functions and struct and types/containers will have their own functions and structs. Golang suggests importing the package whose function and structs that we want to use. This way it removes the over-fetching of packages that the user might not need. Let me know what you think.